modules/nf-core/scramble/clusteranalysis/main.nf

```diff
@@ -4,46 +4,68 @@ process SCRAMBLE_CLUSTERANALYSIS {
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/scramble:1.0.1--h779adbc_1':
-        'biocontainers/scramble:1.0.1--h779adbc_1' }"
+        'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/65/65d3a32dfd347b370e87589189717c75468e6d737b7cee6931e4dae21ce1a9cf/data':
+        'community.wave.seqera.io/library/bioconductor-pwalign_scramble:31d27d3832b0689e' }"
 
     input:
-    tuple val(meta), path(clusters)
-    path fasta
+    tuple val(meta) , path(clusters)
+    tuple val(meta2), path(fasta)
     path mei_ref
 
     output:
-    tuple val(meta), path("*_MEIs.txt")                 , optional:true, emit: meis_tab
-    tuple val(meta), path("*_PredictedDeletions.txt")   , optional:true, emit: dels_tab
-    tuple val(meta), path("*.vcf")                      , optional:true, emit: vcf
-    path "versions.yml"                                 , emit: versions
+    tuple val(meta), path("*_MEIs.txt")              , emit: meis_tab, optional:true
+    tuple val(meta), path("*_PredictedDeletions.txt"), emit: dels_tab, optional:true
+    tuple val(meta), path("*.vcf")                   , emit: vcf     , optional:true
+    path "versions.yml"                              , emit: versions
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
-    def VERSION = '1.0.1' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
+    def VERSION = '1.0.2' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
+
+    def scramble_path = workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1
+        ? "\$CONDA_PREFIX/share/scramble" : "/opt/conda/share/scramble"
```

Comment:
I'm surprise that this work for singularity and docker instead of `/usr/local/share/scramble`, but this might be due to how the container have been built.

Reply:
Don't know, but it does work and `/usr/local/share/scramble` doesn't.